src/components/v5/common/ActionSidebar/partials/RemoveDraftModal/RemoveDraftModal.tsx

```diff
@@ -35,7 +35,7 @@ const RemoveDraftModal: FC<RemoveDraftModalProps> = ({
   }
 
   return (
-    <ModalBase isOpen={isOpen} isFullOnMobile>
+    <ModalBase isOpen={isOpen} isFullOnMobile className="pl-6 pt-6">
```

Comment:
You could use here the existing `hasPadding` property that will set exactly these 2 css classes